build/bin/smoke-riab

```diff
@@ -131,16 +131,63 @@ sleep 60
 
 # make sure that RIAB is running by checking for an onos e2t connection
 ITERATIONS=6
+OAI_RIC_FLAG=false
+OAI_PING_ROUTER_FLAG=false
+OAI_PING_INTERNET_FLAG=false
+OAI_PING_DNS_FLAG=false
 for ((i=1;i<=ITERATIONS;i++))
 do
     if kubectl run cli-check-connections -n riab --rm -i --image onosproject/onos-cli:latest --restart Never --command /usr/local/bin/onos e2t list connections | grep --text E_NB
     then
-        echo RiaB oai smoke test passed!
-        exit 0
+        echo [RiaB OAI smoke test] RIC Control plane is working!
+        OAI_RIC_FLAG=true
+        break
     fi
     sleep 10
 done
 
```

Comment:
You will have to initialize the OAI_PING_ROUTER_FLAG, OAI_PING_INTERNET_FLAG, OAI_PING_DNS_FLAG, and OAI_PING_DNS_FLAG. The script has "set -u" enabled and will exit if you try to read the empty values.

Reply:
Uh, I think they are initialized from 134-137. All flags are set to false before checking. Are those L134-137 something that you mean?

Reply:
haha sorry I didn't see that - it should be fine!